corehq/apps/locations/models.py

```diff
@@ -79,6 +79,14 @@ class LocationType(models.Model):
     administrative = models.BooleanField(default=False)
     shares_cases = models.BooleanField(default=False)
     view_descendants = models.BooleanField(default=False)
+    _expand_from = models.ForeignKey(
+        'self',
+        null=True,
+        related_name='+',
+        db_column='expand_from',
+    )  # levels below this location type that are synced
+    _expand_from_root = models.BooleanField(default=False, db_column='expand_from_root')
+    expand_to = models.ForeignKey('self', null=True, related_name='+')  # levels above this type that are synced
```

Comment:
I think these comments could be clarified a little - this one controls which levels are synced, but `expand_from` concerns more where the expansion starts.